apps/web/playwright/booking/responsiveBooking.e2e.ts

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable playwright/no-conditional-in-test */
+import { loginUser } from "../fixtures/regularBookings";
+import { test } from "../lib/fixtures";
+
+const resolutions = [
+  {
+    width: 1920,
+    height: 1080,
+  },
+  {
+    width: 1280,
+    height: 720,
+  },
+  {
+    width: 640,
+    height: 480,
+  },
+];
+
+test.describe("Booking With All Questions", () => {
+  test.beforeEach(async ({ page, users, bookingPage }) => {
+    await loginUser(users);
+    await page.goto("/event-types");
+    await bookingPage.goToEventType("30 min");
+    await bookingPage.goToTab("event_advanced_tab_title");
+
+    const allQuestions = [
+      "phone",
+      "address",
+      "checkbox",
+      "boolean",
+      "textarea",
+      "multiemail",
+      "multiselect",
+      "number",
+      "radio",
+      "select",
+      "text",
+    ];
+    for (const question of allQuestions) {
```

Comment:
Do we really need to test all of these questions? It's slowing down the E2E suite quite a bit by having this. I want to make sure our value-to-time ratio is good on these. We are already seeing big slow downs in E2E from other tests that have been implemented this way.

Reply:
In this case we are adding as many questions as possible to change the resolution and ensure it is responsive. It won't take long because we are just adding these questions, we will not schedule a meeting after that.

Reply:
But we can remove it if you want 😄